src/Adapters/DiagnosticSource/DiagnosticSource.csproj

```diff
@@ -0,0 +1,44 @@
+﻿
+<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="..\..\..\..\packages\MicroBuild.Core.0.2.0\build\MicroBuild.Core.props" Condition="Exists('..\..\..\..\packages\MicroBuild.Core.0.2.0\build\MicroBuild.Core.props')" />
+
+  <PropertyGroup>
+    <TargetFramework>netstandard1.3</TargetFramework>
```

Comment:
I remember we discussed adding net4.5 target, otherwise users will get bunch of .net core packages in .NET framework project